packages/stacks-svelte/src/components/EmptyState/EmptyState.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import Icon from "../Icon/Icon.svelte";
+    import type { Snippet } from "svelte";
+    import type { ClassValue } from "svelte/elements";
+    import { SpotEmpty } from "@stackoverflow/stacks-icons/spots";
+    interface Props {
+        /**
+         * The optional title to be shown
+         */
+        title?: string;
+
+        /**
+         * Additional CSS classes added to the underlying outer HTML element
+         */
+        class?: ClassValue;
+
+        /**
+         * Snippet containing brief text
+         * explaining the empty state
+         */
+        description: Snippet;
+
+        /**
+         * Optional snippet for actionable buttons
+         */
+        callToAction?: Snippet;
```

Comment:
For consistency across the library, can we call this `actions` like `Notice`/`Toast`?